css/index.css

```diff
@@ -3,45 +3,67 @@
 
 * { 
   line-height: 1.5;
+  
 }
 
 body {
-    background-color: rgb(222,195,224);
-    color: rgb(0,0,0); 
+    background-color: rgb(222, 195, 224);
+    color: rgb(0, 0, 0); 
     font-family: "Times New Roman", serif;
     font-size: 20px;  
     p.normal {
         font-weight: normal;
       } 
-      margin: 0 auto;
+      margin: 0 auto;    
+}
+
+h1 {
+  color: rgb(159, 77, 137); 
+  font-family: "Dancing Script", cursive;
+  font-size: 60px;  
+  font-weight: 900; 
+  margin: 10px;
+  margin-left: 30px;
+  margin-bottom: -20px;
+}
+
+h2 {
+  color: rgb(126, 92, 129); 
+  size: 25px;
+  font-weight: bold; 
 }
- 
- .container {
+
+p {
+letter-spacing: 1.2px;
+}
+
+
+ /* navigation bar*/
+.container-nav {
     background-color: rgb(159, 77, 137);
     overflow: hidden;
     width: 100%;
-    margin: 0 auto;
+    margin: 0 auto;   
   }
 
-  .container a {
+.container-nav a {
     float: right;
     color: rgb(0, 0, 0);
+    letter-spacing: 1px;
     text-align: center;
     padding: 14px 16px;
     text-decoration: none;
-    font-size: 20px;
+    font-size: 20px; 
   }
 
-/* Change the color of links on hover */
-  .container a:hover {
-    background-color: rgb(255, 105, 180);
-    color: black;
+.container-nav a:hover {
+    background-color: rgb(255, 105, 180); /* Change the color of links on hover */
+    color: rgb(0, 0, 0);
   }
   
-  /* Add a color to the active/current link */
-  .container a.active {
-    background-color: rgb(113,55,98);
-    color: rgb(255, 105, 180);
+.container-nav a.active {
+    background-color: rgb(113, 55, 98);
+    color: rgb(255, 105, 180);  /* Add a color to the active/current link */
   }
 
 .nav {
```

Comment:
I'm not sure the `.nav` css  lines do anything currently. It looks `container-nav` controls the position of the nav bar.